app/build.gradle

```diff
@@ -92,6 +94,38 @@ android {
     }
 }
 
+tasks.register('codeCoverage', JacocoReport) {
```

Comment:
chore: I've downloaded coverage report from https://github.com/appunite/Loudius/actions/runs/6614663332?pr=158 and looks like it generates coverage report from test classes instead of code coverage report from the app code itself. 

We should check if application code is covered by tests, not tests themselves.

Reply:
sharp eye 🦅  I did not notice it somehow

Reply:
should be good now 👍

Reply:
actually I see that I need to exclude some files :S so WIP for now